### PR TITLE
site updates

### PR DESCRIPTION
**Pull request by matthidinger**
- Updated/fixed the schema docs based on final v1.0. Added lang, marked title required on Actions, added style to Column, etc.
- Added Partner Status link on Visualizer to make it easier for customers to know when things are ready to use (still need to update the page)
- All JSON snippets switched to tabs for better site formatting
- HTTPS support
- Mobile view for all pages
- Switched to Monaco editor with some IntelliSense working
- Rich previewing in Visualizer for Cortana and Timeline

### CODE REVIEW

File: samples/v1.0/HostConfig/bing.json [hunk 1]
```diff
@@ -1,121 +1,121 @@
...
+	},
+	"containerStyles": {
+		"default": {
+			"backgroundColor": "#FFFFFFFF",
+			"foregroundColors": {
+				"default": {
+					"default": "#FF333333",
+					"subtle": "#EE333333"
+				},
+				"accent": {
```
Comment (dclaux) [inline samples/v1.0/HostConfig/bing.json]: Some HostConfigs like this one are missing the light and dark color definitions.

Comment (matthidinger) [reply samples/v1.0/HostConfig/bing.json]: I'll just delete it, it's old and not used right now

**Comment by matthidinger** [comment]
@dclaux this look good to merge?

**Comment by dclaux** [comment]
Yes, go ahead.

**Pull request merged**